core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -51,6 +52,10 @@ default boolean isKRaftTest() {
 
     Map<Integer, KafkaBroker> brokers();
 
+    default List<KafkaBroker> aliveBrokers() {
```

Comment:
Could it return `Map<Integer, KafkaBroker>` which is same as `brokers`?

Reply:
Done.